### PR TITLE
#167727440 Setup Sequelize

### PR DESCRIPTION
**Pull request by O-Osahon**
### What does this PR do?
- Sets up the application to use Sequelize

### Description of tasks to be completed
- install `sequelize-cli`
- add `sequelize` and `pg-hstore` the application's dependencies
- Add  a `.sequelizerc` file, instructing how Sequelize should be bootstrapped
- Set database credentials in the `.env` file 
- Add migration and "undo migration" scripts to `package.json`

### How should this be manually tested?
-  Clone or download this repository
-  On your terminal, `cd` into the application's directory and run: `git checkout ch-setup-sequelize-167727440`
-  Set the following environment variables in your `.env` file:
```
DEV_DB_USER // database username
DEV_DB_PASSWORD // database password
DEV_DATABASE  //name of your local database
DATABASE_URL  // production database url
```

- If you have not, install [Postgresql](https://www.postgresql.org/download)
- Run `npm run db:migrate`
- Check your database, you should now have a Users' table.
- To undo migration, run `npm run undo:migrate`

### What are the relevant Pivotal Tacker stories?
[#167727440](https://www.pivotaltracker.com/story/show/167727440)

### CODE REVIEW

**Review by parkerthegeniuschild — APPROVED** [review]
Good job !!! If only you would fix your conflicts as soon as you can.

File: src/config/db.config.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const dotenv = require('dotenv');
```
Comment (okoroemeka) [inline src/config/db.config.js:1]: Please use ES6 import here.

File: .sequelizerc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const path = require('path');
```
Comment (okoroemeka) [inline .sequelizerc:1]: Please use ES6 import here.

File: src/database/migrations/20190815042245-create-user.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+'use strict';
```
Comment (okoroemeka) [inline src/database/migrations/20190815042245-create-user.js:1]: Remove strict mode, refactor code to ES6.

File: src/database/migrations/20190815042245-create-user.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('Users');
+  }
+};
```
Comment (okoroemeka) [inline src/database/migrations/20190815042245-create-user.js:33]: A new line needed after the end of this statement.

**Comment by okoroemeka** [comment]
Please squash your commits into one and fix merge conflicts.

File: src/config/db.config.js (hunk 1 above)
Comment (O-Osahon) [reply src/config/db.config.js:1]: ES6 import throws an error. I have tried to fix it using babel as it was specified in the documentation, but it still throws the error. 
I am working on it now though. If nothing seems to work after an hour or two, I will let you know.

**Pull request closed without merging**